modules/passkey/hardhat.config.ts

```diff
@@ -1,15 +1,40 @@
 import '@nomicfoundation/hardhat-toolbox'
+import dotenv from 'dotenv'
 import type { HardhatUserConfig } from 'hardhat/config'
 import 'hardhat-deploy'
 
+// Load environment variables.
+dotenv.config()
+
 const config: HardhatUserConfig = {
   paths: {
     artifacts: 'build/artifacts',
     cache: 'build/cache',
     deploy: 'src/deploy',
     sources: 'contracts',
   },
-  solidity: '0.8.24',
+  networks: {
+    localhost: {
+      url: 'http://localhost:8545',
+      tags: ['dev', 'safe'],
+    },
+    hardhat: {
+      blockGasLimit: 100000000,
```

Comment:
Do we need to increase the block gas limit? That doesn't sound like a good idea.

Reply:
I would remove it unless we need it.

Reply:
> I would remove it unless we need it.

Removing it makes the tests fail:

```
1) P256VerifierLib
       Should return true on valid signature:
     ProviderError: Transaction gas limit is 100000000 and exceeds block gas limit of 30000000
      at EdrProviderWrapper.request (/home/mmv/Projects/Safe/safe-modules/node_modules/hardhat/src/internal/hardhat-network/provider/provider.ts:424:19)
      at async HardhatEthersSigner.sendTransaction (/home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/@nomicfoundation/hardhat-ethers/src/signers.ts:125:18)
      at async ContractFactory.deploy (/home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/ethers/src.ts/contract/factory.ts:111:24)
      at async /home/mmv/Projects/Safe/safe-modules/modules/passkey/test/verifiers/P256VerfierLib.spec.ts:13:25
      at async /home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/hardhat-deploy/src/DeploymentsManager.ts:341:24
      at async Context.<anonymous> (/home/mmv/Projects/Safe/safe-modules/modules/passkey/test/verifiers/P256VerfierLib.spec.ts:21:48)

  2) P256VerifierLib
       Should return false on invalid signature:
     ProviderError: Transaction gas limit is 100000000 and exceeds block gas limit of 30000000
      at EdrProviderWrapper.request (/home/mmv/Projects/Safe/safe-modules/node_modules/hardhat/src/internal/hardhat-network/provider/provider.ts:424:19)
      at async HardhatEthersSigner.sendTransaction (/home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/@nomicfoundation/hardhat-ethers/src/signers.ts:125:18)
      at async ContractFactory.deploy (/home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/ethers/src.ts/contract/factory.ts:111:24)
      at async /home/mmv/Projects/Safe/safe-modules/modules/passkey/test/verifiers/P256VerfierLib.spec.ts:13:25
      at async /home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/hardhat-deploy/src/DeploymentsManager.ts:341:24
      at async Context.<anonymous> (/home/mmv/Projects/Safe/safe-modules/modules/passkey/test/verifiers/P256VerfierLib.spec.ts:31:39)

  3) P256VerifierLib
       Should check for signature signature malleability:
     ProviderError: Transaction gas limit is 100000000 and exceeds block gas limit of 30000000
      at EdrProviderWrapper.request (/home/mmv/Projects/Safe/safe-modules/node_modules/hardhat/src/internal/hardhat-network/provider/provider.ts:424:19)
      at async HardhatEthersSigner.sendTransaction (/home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/@nomicfoundation/hardhat-ethers/src/signers.ts:125:18)
      at async ContractFactory.deploy (/home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/ethers/src.ts/contract/factory.ts:111:24)
      at async /home/mmv/Projects/Safe/safe-modules/modules/passkey/test/verifiers/P256VerfierLib.spec.ts:13:25
      at async /home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/hardhat-deploy/src/DeploymentsManager.ts:341:24
      at async Context.<anonymous> (/home/mmv/Projects/Safe/safe-modules/modules/passkey/test/verifiers/P256VerfierLib.spec.ts:37:48)

  4) P256VerifierLib
       Should return false for misbehaving verifiers:
     ProviderError: Transaction gas limit is 100000000 and exceeds block gas limit of 30000000
      at EdrProviderWrapper.request (/home/mmv/Projects/Safe/safe-modules/node_modules/hardhat/src/internal/hardhat-network/provider/provider.ts:424:19)
      at async HardhatEthersSigner.sendTransaction (/home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/@nomicfoundation/hardhat-ethers/src/signers.ts:125:18)
      at async ContractFactory.deploy (/home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/ethers/src.ts/contract/factory.ts:111:24)
      at async /home/mmv/Projects/Safe/safe-modules/modules/passkey/test/verifiers/P256VerfierLib.spec.ts:13:25
      at async /home/mmv/Projects/Safe/safe-modules/modules/passkey/node_modules/hardhat-deploy/src/DeploymentsManager.ts:341:24
      at async Context.<anonymous> (/home/mmv/Projects/Safe/safe-modules/modules/passkey/test/verifiers/P256VerfierLib.spec.ts:48:38)```